### ISSUE TITLE
Resolve WCAG 2.0 issues from second AbilityNet report

### ISSUE THREAD

**Issue by anjackson** [ukwa/ukwa-ui#246]
As well as these issues from #245 :

- [ ] WCAG-020 Low colour contrast (1,6,7,9,10) 'featured', grey in forms, but also red alerts.
- [ ] WCAG-021 Focus not visible on interactive elements (now red outline, but not sure what to do for high contrast)

We need to resolve the issues from the second AbilityNet report that covers the `search` page.

**Comment by nicolabingham** [comment]
@min2ha I need to book in the AbilityNet recheck soon. Can we discuss getting these two outstanding issues fixed please?

**Comment by min2ha** [comment]
@nicolabingham I'm in progress with audit of what we need to do. I'll present report next week.

**Comment by min2ha** [comment]
MacOS problem: Safari and Firefox users affected.
(Mozilla/FF is being true to operating system settings in Mac OS.)

_There are two distinct ways around this on the user side. Both seem to work:_

- In _System Preferences → Keyboard_, in the _Shortcuts_ pane, check the “_all controls_” radio at the bottom.

- In Firefox, type "_about:config_" in the URL bar. There is no _accessibility.tabfocus_ preference on the mac, so you'll have to make one. Right click in the window, create a new "integer" pref, and set it to 7.

**_Neither of these are terribly obvious. Also, neither of these are a server-side solution for developers, which is frustrating._**


https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox

**Comment by min2ha** [comment]
Screen Readers/extensions used to test:

- NVDA
- ChromeVox Classic Extension
- ORCA
